systems/velero/values.tmpl.yaml

```diff
@@ -35,5 +35,16 @@ velero:
       bucket: {{ .Requirements.storage.backup.url | removeScheme | quote }}
       config:
         storageAccount: {{ .Requirements.velero.serviceAccount | quote }}
+{{- else if eq .Requirements.cluster.provider "iks" }}
+  snapshotsEnabled: false
+  configuration:
+    provider: aws
```

Comment:
Why is the provider `aws` since you are trying to enable it for `iks`?

Reply:
Thanks for the review. Basically, IKS is not directly supported by Velero https://velero.io/docs/v1.1.0/support-matrix/ but, since IBM Cloud Object Storage is compatible with S3 Apis, then "aws" configuration - with proper URL set - is recommended (https://velero.io/docs/v1.1.0/ibm-config/). We are using this configuration with our IKS clusters.

Reply:
Thanks! It makes sense now.